### ISSUE TITLE
tf.profiler provides incompatible shape for ssd mobilenet V2 object detection model

### ISSUE THREAD

**Issue by subhamvs** [tensorflow/models#5416]
### System information
- **What is the top-level directory of the model you are using**: ssd_mobilenet_v2
- **Have I written custom code (as opposed to using a stock example script provided in TensorFlow)**: Yes
- **OS Platform and Distribution (e.g., Linux Ubuntu 16.04)**:  Linux Ubuntu 16.04
- **TensorFlow installed from (source or binary)**: Using pip
- **TensorFlow version (use command below)**: 1.10.0
- **Bazel version (if compiling from source)**: No
- **CUDA/cuDNN version**: 9.0/7.1
- **GPU model and memory**: Geforce GTX 1080 Ti / 11171 MB

### Describe the problem
I am facing "incompatible shape" issue while profiling tensorflow objection detection model. 
Node FeatureExtractor/MobilenetV2/Conv/Conv2D incompatible shapes: Shapes (?, ?, ?, 32) and (1, 32, 256, 256) are not compatible.
Node FeatureExtractor/MobilenetV2/Conv/Relu6 incompatible shapes: Shapes (?, ?, ?, 32) and (1, 32, 256, 256) are not compatible.
Node FeatureExtractor/MobilenetV2/expanded_conv/depthwise/depthwise incompatible shapes: Shapes (?, ?, ?, 32) and (1, 32, 256, 256) are not compatible.
Node FeatureExtractor/MobilenetV2/expanded_conv/depthwise/Relu6 incompatible shapes: Shapes (?, ?, ?, 32) and (1, 32, 256, 256) are not compatible.
Node FeatureExtractor/MobilenetV2/expanded_conv/project/Conv2D incompatible shapes: Shapes (?, ?, ?, 16) and (1, 16, 256, 256) are not compatible.
...
Basically, I want to get flip flops of the model.
How to fix the issue? Is there any other way to get flops of the model?

**Comment by tensorflowbutler** [comment]
Thank you for your post. We noticed you have not filled out the following field in the issue template. Could you update them if they are relevant in your case, or leave them as N/A? Thanks.
Exact command to reproduce

**Comment by subhamvs** [comment]
I took frozen pb file from **ssd_mobilenet_v2_coco** 
[https://github.com/tensorflow/models/blob/master/research/object_detection/g3doc/detection_model_zoo.md](url)
# Code
`
import tensorflow as tf
import sys

frozen_checkpoint = sys.argv[1]
with tf.gfile.FastGFile(frozen_checkpoint, 'rb') as f:
    graph_def = tf.GraphDef()
    graph_def.ParseFromString(f.read())

with tf.Graph().as_default() as graph:
    tf.import_graph_def(graph_def, name='')

run_metadata = tf.RunMetadata()
opts = tf.profiler.ProfileOptionBuilder.float_operation()    
flops = tf.profiler.profile(graph, run_meta=run_metadata, cmd='op', options=opts)
print (flops.total_float_ops)
`

Please correct me if I am doing wrong.

**Comment by tensorflowbutler** [comment]
Hi There,
 We are checking to see if you still need help on this, as this seems to be considerably old issue. Please update this issue with the latest information, code snippet to reproduce your issue and error you are seeing.
 If we don't hear from you in the next 7 days, this issue will be closed automatically. If you don't need help on this issue any more, please consider closing this.
